src/main/java/org/icgc_argo/wes/argo/api/config/ElasticsearchConfig.java

```diff
@@ -21,12 +21,12 @@ public ElasticsearchConfig(@NonNull ElasticsearchProperties properties) {
   @Bean
   public RestHighLevelClient restHighLevelClient() {
     return new RestHighLevelClient(
-            RestClient.builder(new HttpHost(properties.getHost(), properties.getPort()))
```

Comment:
reference the latest changes in search and relay, we need to include the ability to login and/or use https
(https://github.com/icgc-argo/workflow-search/pull/35/files)